src/prefect/utilities/feature_flags.py

```diff
@@ -0,0 +1,171 @@
+"""
+This module contains code related to feature flagging.
```

Comment:
> contains code

What else does a module contain?  :P 

How about something like:

```suggestion
Support for feature flags enabling experimental behavior in Prefect.
```